master/cache.lua

```diff
@@ -257,7 +260,7 @@ end
 local function fetch_and_store_state_mesos()
     -- Fetch state JSON summary from Mesos. If successful, store to SHM cache.
     -- Expected to run within lock context.
-    local mesosRes, err = request("leader.mesos", 5050, "/master/state-summary", false)
+    local mesosRes, err = request("http://leader.mesos:5050" .. "/master/state-summary", false)
```

Comment:
I knew that you will point this out :D 

Try diffing EE and Open. Writing it this way will make it a bit easier for me to merge it with EE at some point.